src/components/ReactAutoSuggestHelper.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Autosuggest from 'react-autosuggest';
+import match from 'autosuggest-highlight/match';
+import parse from 'autosuggest-highlight/parse';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import { API } from 'aws-amplify';
+
+export interface OptionType {
+	label: string;
+}
+
+export function renderInputComponent(inputProps: any) {
+	const { classes, inputRef = () => {}, ref, ...other } = inputProps;
+
+	return (
+		<TextField
+			fullWidth
+			InputProps={{
+				inputRef: node => {
+					ref(node);
+					inputRef(node);
+				},
+				classes: {
+					input: classes.input
+				}
+			}}
+			{...other}
+		/>
+	);
+}
+
+export const performSearch = async (query: string, registryName: string) => {
+	let response = await API.get(
+		'entity',
+		'/registry/' + registryName + '/search?query=' + query,
+		null
+	);
+	const results = response.map((item: any) => {
+		return {
+			id: item.id,
+			label: item.preferredLabel[0].value
```

Comment:
Does [0] imply Bokmål?

Reply:
no, but will change it to choose norwegian first, and if id does not exist then it will pick the first one